docs/guis/multiturn/excitation.md

```diff
@@ -0,0 +1,207 @@
+# Performing Beam Excitation
+
+Once [all the checks](safety.md) have been performed, one can start measurements.
+In the Multiturn GUI one can perform beam excitation with either the AC Dipole or the ADT, and both procedures are very similar.
+Select either the `ACDipole` or `ADTACDipole` tab at the top of the GUI, depending on the desired excitation device.
+
+## Common Settings
+
+The following settings are set identically for both the AC Dipole and the ADT, in the left-hand side of the GUI:
+
+### Kick Groups
+
+Before exciting the beam, one should select or create a kick group.
+This is done by clicking the ++"Select Active group"++ button in the top left corner of the GUI, which will open the following dialog:
```

Comment:
Explain why.

This collects all following kicks under a single name and makes it easier in the future to simultaneously load related kicks at once identify what these kicks were used for. 
Therefore it is important to make sure the given name is clear and explicit!

??? tip "Kickgroups in json"
    Each kick group also has a corresponding `.json` file in `/user/slops/data/LHC_DATA/OP_DATA/Betabeat/KickGroups/MULTITURN_ACQ_GROUPS` in which the paths to the acquired turn-by-turn data and their individual `.json` files containing information about the excitation parameter is stored.
    
    See also the [PyLHC tool for KickGroups](https://github.com/pylhc/PyLHC/blob/master/pylhc/kickgroups.py)